lib/plugins/Label_H1_OHMA.ts

```diff
@@ -28,14 +28,20 @@ export class Label_H1_OHMA extends DecoderPlugin {
       decompress.flush(zlib.constants.Z_SYNC_FLUSH);
       const result = decompress.read();
       const jsonText = result.toString();
-      const json = JSON.parse(jsonText);
+      
       let formattedMsg;
+      let jsonMessage;
+      try {
+        jsonMessage = JSON.parse(jsonText).message;
+      } catch {
+        jsonMessage = jsonText;
+      }
```

Comment:
yuck - open to better ideas

Reply:
Yeah, not amazing, but I can't think of a better way to approach at the moment.